raiden/ui/app.py

```diff
@@ -238,23 +238,8 @@ def run_app(
         eth_rpc_endpoint = f'http://{eth_rpc_endpoint}'
 
     web3 = _setup_web3(eth_rpc_endpoint)
-
-    rpc_client = JSONRPCClient(
-        web3,
-        privatekey_bin,
-        gas_price_strategy=gas_price,
-        block_num_confirmations=DEFAULT_NUMBER_OF_BLOCK_CONFIRMATIONS,
-        uses_infura='infura.io' in eth_rpc_endpoint,
-    )
-
-    blockchain_service = BlockChainService(
-        jsonrpc_client=rpc_client,
-        # Not giving the contract manager here, but injecting it later
-        # since we first need blockchain service to calculate the network id
-    )
-
     given_network_id = network_id
-    node_network_id = blockchain_service.network_id
+    node_network_id = int(web3.version.network)
```

Comment:
nice! Just use web3 directly here to get rid of the ugly partial initialization. Thanks.